CHANGELOG.md

```diff
@@ -44,6 +47,18 @@
   - [changelog](https://github.com/getsentry/sentry-cli/blob/master/CHANGELOG.md#2421)
   - [diff](https://github.com/getsentry/sentry-cli/compare/2.41.1...2.42.1)
 
+
+### Major Changes
+
+- Remove autoSessionTracking option
```

Comment:
The Browser SDK now recommends directly using https://github.com/getsentry/sentry-javascript/blob/defeaaf99ff6180be714ae770ce1ed0da6243676/packages/browser/src/integrations/browsersession.ts#L12

RN SDK has also `enableAutoSessionTracking` https://github.com/getsentry/sentry-react-native/blob/f8289de4acf2755f5a9180dd33772fc38d59c7ea/packages/core/src/js/options.ts#L45

`autoSessionTracking` and `enableAutoSessionTracking` were two unrelated flags.

We should make it clear to the users.

Reply:
In regard to this, should we remove the integration `autoSessionTracking` when running natively?

Reply:
Yes, makes sense, let's add the `browserSessionIntegration` when `enableAutioSessionTracking` and we are running in a browser.

Reply:
We can also do this in a follow up PR.